docs/user/troubleshooting/first-troubleshooting-guide.md

```diff
@@ -0,0 +1,21 @@
+# Istio troubleshooting guide
+
+If you are experiencing issues with the cluster, you can follow the steps below to troubleshoot the problem before you contact us. This guide is designed to help you identify the root cause of the problem which might not necessarily be related to the Istio itself but rather to the misconfiguration of the Istio Custom Resources or other network related resources.
+
+## Try to go through the following steps:
```

Comment:
I'd not have this as steps. I'd rather expect that we find different topics, e.g. Network connectivity, Sidecar Injection, Istio CR. And we add scenarios or use-cases and list steps to check how this might be resolved.
e.g.
## Network connectivity
### Ingress/Egress traffic is not working
1. Check NetworkPolicies
2.  ...

## Sidecar Injection
### Proxy SIdecar is running an old Proxy version
1. Check if Istio CR is in Warning State, ...
2. ...